metron-deployment/playbooks/metron_install.yml

```diff
@@ -74,47 +83,115 @@
   tags:
     - mysql-client
 
-- hosts: sensors
-  become: true
-  roles:
-    - ambari_gather_facts
-    - { role: tap_interface, when: install_tap | default(False) == True }
-    - { role: pycapa, when: install_pycapa | default(True) == True }
-    - { role: bro, when: install_bro | default(True) == True }
-    - { role: flume,  when: install_snort | default(True) == True }
-    - { role: snort , when: install_snort | default(True) == True }
-    - { role: yaf, when: install_yaf | default(True) == True }
-    - { role: pcap_replay, when: install_pcap_replay | default(False) == True }
-    - { role: sensor-test-mode, when: sensor_test_mode | default(False) == True }
-  tags:
-      - sensors
-
+#
+# parsers, enrichment, and indexing topologies
+#
 - hosts: enrichment
   become: true
   roles:
     - role: metron_streaming
   tags:
     - enrichment
 
+#
+# user interface
+#
 - hosts: pcap_server
   become: true
   roles:
-    - { role: metron_pcapservice, when: install_elasticsearch | default(True) == True }
+    - role: metron_pcapservice
   tags:
-    - pcap_service
+    - pcap-service
 
 - hosts: web
   become: true
   roles:
-    - { role: kibana, when: install_elasticsearch | default(True) == True }
+    - role: kibana
   tags:
-    - web
+    - kibana
 
+#
+# sensors
+#
+- hosts: sensors
+  become: true
+  roles:
+    - role: ambari_gather_facts
+    - role: tap_interface
+  tags:
+    - tap
+
+- hosts: sensors
+  become: true
+  roles:
+    - role: ambari_gather_facts
+    - role: pycapa
+  tags:
+    - pycapa
+
+- hosts: sensors
+  become: true
+  roles:
+    - role: ambari_gather_facts
+    - role: bro
+  tags:
+    - bro
+
+- hosts: sensors
+  become: true
+  roles:
+    - role: ambari_gather_facts
+    - role: flume
+  tags:
+    - flume
+
+- hosts: sensors
+  become: true
+  roles:
+    - role: ambari_gather_facts
+    - role: snort
+  tags:
+    - snort
+
+- hosts: sensors
+  become: true
+  roles:
+    - role: ambari_gather_facts
+    - role: yaf
+  tags:
+    - yaf
+
+- hosts: sensors
+  become: true
+  roles:
+    - role: ambari_gather_facts
+    - role: pcap_replay
+  tags:
+    - pcap-replay
+
+- hosts: sensors
+  become: true
+  roles:
+    - role: ambari_gather_facts
+    - role: sensor-test-mode
+  tags:
+    - sensor-test-mode
+
```

Comment:
I think I remembered my comment from last week- you may be able to combine these into a single hosts/roles group by using the following construct:
```yaml
hosts: sensors
become: true
tags: sensors
roles:
     -{ role: ambari_gather_facts, tags: [ 'always'] }
    - { role: tap_interface, tags: ['tap'] }
    - { role: pycapa, tags: ['pycapa'] }
    - { role: bro, tags: ['bro'] }
    - { role: flume,  tags: ['snort','flume'] }
    - { role: snort , tags: ['snort'] }
    - { role: yaf, tags: ['yaf'] }
    - { role: pcap_replay, tags: ['pcap-replay'] }
    - { role: sensor-test-mode, tags: ['sensor-test-mode'] }
```
This is per the docs, so ymmv. But if it works as advertised, I think it's a little bit more readable. Otherwise do what you got to do.